admin/test/integration/workarea/admin/jump_to_integration_test.rb

```diff
@@ -127,7 +127,7 @@ def test_finding_content_assets
 
         results = JSON.parse(response.body)['results']
         assert_equal(1, results.length)
-        assert_equal('Test Asset - file.jpg', results.first['label'])
+        assert_equal('Test Asset - product_image.jpg', results.first['label'])
```

Comment:
we fixed it lol

Reply:
it was changed because the factory now has the filename.